docs/about/team/the-software-development-team.md

```diff
@@ -4,6 +4,26 @@ sidebar_position: 2
 
 # The Software development team
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon)
+
+Hello! My name is Maxence Guindon, and I am a backend developer at the Canadian
+Food Inspection Agency AI-Lab.
```

Comment:
this document seems to have as many way to name our lab than they are people. can we get some consistency here?

Reply:
yes

Reply:
Done